### PR TITLE
Kalmroth lab4

### PR DESCRIPTION
**Pull request by krille71**

### CODE REVIEW

File: Lab-4/kalmroth_lab4.ipynb [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "WeFO9QMeUOAu",
+        "colab_type": "text"
+      },
+      "source": [
+        "# **Methods**"
```
Comment (ajlace) [inline Lab-4/kalmroth_lab4.ipynb:129]: Add either an explanation to how you got the different values for A and b, or include a reference to where these were found.

File: Lab-4/kalmroth_lab4.ipynb [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SsQLT38gVbn_",
+        "colab_type": "text"
+      },
+      "source": [
+        "# **Results**"
```
Comment (ajlace) [inline Lab-4/kalmroth_lab4.ipynb:203]: Add a plot showing exact solution vs your approximation.

**Review by ajlace — CHANGES_REQUESTED** [review]
Lab 4: Consider adding a more elaborate discussion of the results/lab in the future. Other than the changes requested, good job.

**Comment by krille71** [comment]
Should be fixed now @ajlace

**Review by ajlace — APPROVED** [review]

**Pull request merged**